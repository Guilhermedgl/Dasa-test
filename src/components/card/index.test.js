import React from 'react';
import { create } from 'react-test-renderer';
import Card from './index';

describe('<Card />', () => {
  it('should render props.name', () => {
    const pokemon = {
      name: 'bulbasaur'
    }
    const component = create(<Card pokemon={pokemon} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});