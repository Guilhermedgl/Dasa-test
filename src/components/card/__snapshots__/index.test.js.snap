// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render props.name 1`] = `
<div
  className="card"
>
  <p
    className="name"
  >
    bulbasaur
  </p>
</div>
`;
